### PR TITLE
Release 1.2.3

### PR DESCRIPTION
**Pull request by owenpearson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [v1.2.3](https://github.com/ably/ably-ruby/tree/v1.2.3)
+
+[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.2.2...v1.2.3)
+
+**Closed issues:**
+
+- Incorrect ProtocolMessage\#connection\_details object \(overwrites original connection\_details send on CONNECTED state\) [\#377](https://github.com/ably/ably-ruby/issues/377)
```
Comment (QuintinWillison) [inline CHANGELOG.md:9]: I think this issue needs to be labelled `bug` and so this section should perhaps be entitled 'Fixed Bugs'.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+
+**Merged pull requests:**
```
Comment (QuintinWillison) [inline CHANGELOG.md:11]: I would thin out this section to remove issues that don't directly improve the customer experience of using this SDK... e.g. remove test specs and test observability.

**Review by QuintinWillison — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (owenpearson) [reply CHANGELOG.md:13]: done

File: CHANGELOG.md (hunk 2 above)
Comment (owenpearson) [reply CHANGELOG.md:15]: Yeah good point, I've removed the fluff and structured it a bit better now

**Review by stmoreau — APPROVED** [review]
LGTM!

**Review by QuintinWillison — APPROVED** [review]

**Pull request merged**